images/s6_assets/template_nginx.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import os
+import django
+from django.core.exceptions import AppRegistryNotReady, ImproperlyConfigured
+
+from jinja2 import Template
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Create Pulp's nginx conf file based on current settings.",
+    )
+    parser.add_argument("template_file", type=open)
+    parser.add_argument("output_file", type=argparse.FileType("w"))
+    args = parser.parse_args()
+
+    https = os.getenv("PULP_HTTPS", "false")
+    values = {
+        "https": https.lower() in ("true", "t", "yes", "y", "1"),
```

Comment:
Are these values documented anywhere?
Do _all_ the other places recognize them in the same way?

Reply:
No, previously we were only checking for 'true' after lower-casing.